Tests/BuildTests/BuildPlanTests.swift

```diff
@@ -1714,7 +1714,6 @@ final class BuildPlanTests: XCTestCase {
             "-Xlinker", "-bundle",
             "-Xlinker", "-rpath", "-Xlinker", "@loader_path/../../../",
             "@\(buildPath.appending(components: "PkgPackageTests.product", "Objects.LinkFileList"))",
-            "-Xlinker", "-rpath", "-Xlinker", "/fake/path/lib/swift-5.5/macosx",
```

Comment:
The test was always wrong here because it was working off the previously incorrect implementation which set this rpath based on the minimum deployment target of the *package* instead of the individual target.